app/src/main/java/com/futurice/freesound/feature/search/SearchFragment.java

```diff
@@ -70,6 +71,17 @@ public void bind(@NonNull final CompositeDisposable disposables) {
                                        .observeOn(mainThread())
                                        .subscribe(SearchFragment.this::handleResults,
                                                   e -> Timber.e(e, "Error setting Sound items")));
+            disposables.add(viewModel().getSearchErrors()
+                    .subscribeOn(Schedulers.computation())
+                    .observeOn(mainThread())
+                    .subscribe(throwableOption -> {
+                                if (throwableOption.isSome()) {
+                                    ((SearchActivity) getActivity()).showSnackbar("There was an error searching");
```

Comment:
Do we need to have the snackbar in the activity? The fragment does not need to know in what kind of activity it is in.
We could simply have a snackbar here